deps.edn

```diff
@@ -1,29 +1,15 @@
-{:paths ["resources" "src/cljc" "src/cljs" "src/clj"]
- :deps {org.clojure/clojure #:mvn{:version "1.10.1"}
-        org.clojure/clojurescript
-        {:exclusions [com.google.javascript/closure-compiler-unshaded
-                      org.clojure/google-closure-library
-                      org.clojure/google-closure-library-third-party]
-         :mvn/version "1.10.597"}
-        seancorfield/clj-new {:mvn/version "1.0.199"}
-        re-com/re-com #:mvn{:version "2.7.0"}
-        re-frame/re-frame #:mvn{:version "0.11.0"}
-        reagent/reagent #:mvn{:version "0.9.1"}
-        thheller/shadow-cljs #:mvn{:version "2.8.96"}}
- :aliases {:dev {:extra-paths ["dev-resources" "classes"]
-                 :extra-deps {org.clojure/clojure {:mvn/version "1.10.3"}
-                              org.clojure/clojurescript
+{:paths ["src/cljc"]
+ :aliases {:dev-frontend {:extra-paths ["src/cljs" "resources"]
+                 :extra-deps {org.clojure/clojurescript
                               {:exclusions [com.google.javascript/closure-compiler-unshaded
                                             org.clojure/google-closure-library
                                             org.clojure/google-closure-library-third-party]
                                :mvn/version "1.11.4"}
-                              seancorfield/clj-new {:mvn/version "1.0.199"}
-                              re-com/re-com #:mvn{:version "2.7.0"}
                               re-frame/re-frame #:mvn{:version "0.11.0"}
                               reagent/reagent #:mvn{:version "0.9.1"}
                               thheller/shadow-cljs {:mvn/version "2.18.0"}
                               clj-kondo/clj-kondo {:mvn/version "2022.04.25"}
-                              day8.re-frame/re-frame-10x {:mvn/version "0.7.0"} ;;do we need this?
+                              day8.re-frame/re-frame-10x {:mvn/version "0.7.0"}
```

Comment:
You did add this in another commit in the same pull request. You should remove that using a fixup commit for the single line.